lib/Models/registerCatalogMembers.js

```diff
@@ -177,11 +177,21 @@ var registerCatalogMembers = function() {
     WebFeatureServiceCatalogGroup,
     true
   );
+  createCatalogItemFromUrl.register(
+    matchesUrl(/\=wfs/i),
+    WebFeatureServiceCatalogGroup,
+    true
+  );
   createCatalogItemFromUrl.register(
     matchesUrl(/\/wms/i),
     WebMapServiceCatalogGroup,
     true
   );
+  createCatalogItemFromUrl.register(
+    matchesUrl(/\=wms/i),
```

Comment:
Same as above, I think it would be preferable to use a single regex to capture both options.